### ISSUE TITLE
Can't insert internal wiki links while this extension is enabled

### ISSUE THREAD

**Issue by rrakso** [syntaxseed/dokucrypt2#12]
This is happening, when I try to insert a link.
![image](https://user-images.githubusercontent.com/23143274/118531154-d0e18880-b745-11eb-852e-976270605187.png)

**Comment by syntaxseed** [comment]
Hello.
Which version of DokuWiki are you using?

Can you confirm.... if you disable just this plugin, it works?

What happens if all plugins are disabled EXCEPT this one.. .does the issue still happen?

Thanks!

**Comment by syntaxseed** [comment]
Also... does this happen only on pages that have an encrypted block.... or on all pages?

**Comment by syntaxseed** [comment]
Nevermind... I can reproduce it myself.
Huh. I have literally never used the internal link button. lol

I will look into this! Thanks @rrakso.

**Comment by syntaxseed** [comment]
Well, this has been a rabbit hole.

For reference in the future:

- The plugin puts the wiki text into a hidden input and then moves the original textarea out of the form. (See DokuCrypt2's script.js ~ line 57). This is to try to make sure that the unencrypted text isn't accidentally submitted.

- The moving of the text area out of the form makes the toolbar link button unable to find it and therefor `dw_linkwiz.textArea.form.id.value` is Null.

- The moving of the textarea is done when the edit form is initialized. 

- I changed it so that it is now moved just before submission.

It is possible that if a Save is aborted for some reason, the link button will be broken again... but this is as fixed as I'm going to get it at this time.

Issuing a new release of the plugin with these changes soon.
